doc/lsst.resources/index.rst

```diff
@@ -31,3 +31,9 @@ Python API reference
 
 .. automodapi:: lsst.resources
    :no-main-docstr:
+
+.. automodapi:: lsst.resources.utils
+   :no-main-docstr:
+
+.. automodapi:: lsst.resources._resourceHandles._baseResourceHandle
+   :no-main-docstr:
```

Comment:
I'm surprised this directive doesn't put those private namespaces into the docs.  Or does it actually do that, and it's still the least bad way to get the docs for the classes in at all?

Reply:
They do appear in the docs in their own subsection. The problem is that if you specify private base classes then none of the links from your classes resolve any where unless sphinx has built the docs for the private base class. Maybe there is a directive here that would let the class docs exist but not list them on the main page.

Reply:
Would propagating just the relevant class up to the package namespace via `__all__` work?  Not sure it'd actually be better.

Reply:
Yes. These aren't exported by default so they aren't indexed. The automodapi forces them to be rendered so that they can be cross-referenced. I don't think we really want to have a huge public interface in the python layer to make sphinx's job easier. Maybe we can create a subpage for "private interfaces" and then automodapi them there.

Reply:
I like the "private interfaces" (or maybe "internal interfaces"?) subpage idea.

Reply:
Done that. It's not quite how I'd want it (since it currently shows a list of internal modules on the main page) but better than it was.